src/booking/Offers/OfferTypes.ts

```diff
@@ -164,82 +164,123 @@ export interface OfferAvailableServiceBaggageMetadata {
+   * The list of segment `id`s the service applies to. If you add this
+   * service to an order it will apply to all the segments in this list. For
+   * services where the type is `baggage`, depending on the airline, this
+   * list includes all the segments of all slices or all the segments of a
+   * single slice.
    */
   segment_ids: string[]
 
   /**
-   * The total price of the service for all passengers and segments it applies to, including taxes
+   * The total price of the service for all passengers and segments it
+   * applies to, including taxes. This price is for a single unit of the
+   * service.
    */
   total_amount: string
 
   /**
-   * The currency of the `total_amount`, as an ISO 4217 currency code
+   * The currency of the `total_amount`, as an [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217)
+   * currency code. It will match your organisation's billing currency unless
+   * you’re using Duffel as an accredited IATA agent, in which case it will be
+   * in the currency provided by the airline (which will usually be based on the
+   * country where your IATA agency is registered).
    */
   total_currency: string
+}
+
+export interface OfferAvailableServiceBaggage
+  extends OfferAvailableServiceCommon {
+  /**
+   * The metadata varies by the type of service. It includes further data
+   * about the service. For example, for baggages, it may have data about
+   * size and weight restrictions.
+   */
+  metadata: OfferAvailableServiceBaggageMetadata
+
+  /**
+   * The type of the service.
+   */
+  type: 'baggage'
+}
+
+export interface OfferAvailableServiceCFAR extends OfferAvailableServiceCommon {
+  /**
+   * The metadata varies by the type of service. It includes further data
+   * about the service. For example, for baggages, it may have data about
+   * size and weight restrictions.
+   */
+  metadata: OfferAvailableServiceCFARMetadata
 
   /**
    * The type of the service.
-   * For now we only return services of type baggage but we will return other types in the future.
-   * We won't consider adding new service types a break change.
    */
-  type: T_ServiceType
+  type: 'cancel_for_any_reason'
+}
+
+export type OfferAvailableService =
+  | OfferAvailableServiceBaggage
+  | OfferAvailableServiceCFAR
+
+export interface PaymentRequirements {
+  /**
+   *  The ISO 8601 datetime by which you must pay for this order.
+   * At this time, if still unpaid, the reserved space on the flight(s)
+   * will be released and you will have to create a new order.
+   * This will be null only for orders where `awaiting_payment` is `false`.
+   */
+  payment_required_by?: string | null
+  /**
+   *  The ISO 8601 datetime at which the price associated
+   * with the order will no longer be guaranteed by the airline
+   * and the order will need to be repriced before payment.
+   * This can be null when there is no price guarantee.
+   */
+  price_guarantee_expires_at?: string | null
```

Comment:
nit: we probably don't need these `?`s right? I think we can guarantee that these fields will be included in the response.

Reply:
oh I just saw your above comment. Feel free to leave this for a separate PR if you want!

Reply:
Done!